core/database.py

```
@@ -161,22 +168,25 @@ async def store_message(
 
     try:
         async with pool.acquire() as conn:
-            await conn.execute(
+            result = await conn.execute(
                 """
                 DELETE FROM messages WHERE message_id = $1
                 """,
                 message_id,
             )
+            deleted_count = int(result.split()[-1]) if result else 0
             logger.debug(f"Deleted message {message_id} from database")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)